google/cloud/firestore_v1/bulk_writer.py

```diff
@@ -213,19 +310,37 @@ def __init__(
 
         self._ensure_executor()
 
-    def _reset_batch(self) -> BulkWriteBatch:
-        self._batch = BulkWriteBatch(self._client)
-        return self._batch
+    @staticmethod
+    def _default_on_batch(
+        batch: BulkWriteBatch, response: BatchWriteResponse, bw: "BulkWriter",
```

Comment:
`bw` doesn't seem like the pythonic name, though it isn't awful. It seems to be the name of a writer?